Core/Model/ArticuloCombinacion.php

```diff
@@ -175,39 +171,198 @@ public function clear()
 
     public function test()
     {
-        if ($this->codigo === null) {
-            $this->codigo = (string) $this->getNewCodigo();
+        $sql = 'SELECT * FROM ' . static::tableName() . ' WHERE codigo = ' . $this->dataBase->var2str($cod) . ';';
```

Comment:
Esté código en la función test no tiene mucho sentido. Debe ser solamente:

if ($this->codigo === null) {
            $this->codigo = (string) $this->getNewCodigo();
        }

Reply:
y un return true;